src/main/java/uk/gov/companieshouse/api/accounts/model/entity/AccountDataEntity.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.api.accounts.model.entity;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.GenerateEtagUtil;
+import uk.gov.companieshouse.api.accounts.Kind;
+import uk.gov.companieshouse.api.accounts.LinkType;
+
+public class AccountDataEntity {
+
+    @Field("id")
+    private String id;
+
+    @NotNull
+    @Field("period_end_on")
+    private LocalDate periodEndOn;
+
+    private Map<String, String> links = new HashMap<>();
+
+    private String kind;
+
+    private String etag;
+
+    protected AccountDataEntity build(String id){
+        this.id = id;
+        this.etag = GenerateEtagUtil.generateEtag();
+        this.kind = Kind.ACCOUNT.getValue();
+
+        Map<String, String> links = new HashMap<>();
+        links.put(LinkType.SELF.getLink(), "links/accounts");
```

Comment:
Why has this been hardcoded to "links/accounts"?

Reply:
It's a default value current as we dont know what the format of the links will be - the actual implementation will depend alot of the discussion with the architects

Reply:
We shouldn't be hardcoding temp values. They could get missed and accidentally released.

Reply:
It won't get missed, it is the thing Steve is working on for the Small-full accounts PR. This will be the first thing that we do when that discussion has been had.

Reply:
It should be removed and added when Steve implements his part.

Reply:
done